test/driver.cpp

```diff
@@ -145,9 +146,46 @@ void processCmdLine(std::vector<std::string_view> arguments)
    }
    catch (const std::string & msg)
    {
-      std::cout << msg << '\n'; //TODO: print in appropriate stream based in cmd-line
+      (outputFilename.empty() ? std::cout : fileOut )<< msg << '\n';
    }
 
+   //if print summary option is enabled
    if (printSummary)
       summarizeTests();
-}
\ No newline at end of file
+}
+
+//Replace all instances of a substring with a different substring
```

Comment:
Comment does not need to begin with upper case letter: no need to treat comments as sentences

```suggestion
//replace all instances of a substring with a different substring
```